plugins/metas.ts

```diff
@@ -69,8 +69,8 @@ export default function (userOptions?: Partial<Options>) {
 
       const { document } = page;
       const url = site.url(page.data.url as string, true);
-      const icon = metas.icon ? site.url(metas.icon, true) : undefined;
-      const image = metas.image ? site.url(metas.image, true) : undefined;
+      const icon = metas.icon ? site.url(new URL(metas.icon, url).href, true) : undefined;
+      const image = metas.image ? site.url(new URL(metas.image, url).href, true) : undefined;
```

Comment:
I think `metas.images` should be passed to `site.url()` first, and then resolve with URL.

```suggestion
      const image = metas.image ? new URL(site.url(metas.image), url).href : undefined;
```

Reply:
You got it!